app/forms/schools/add_participant_form.rb

```diff
@@ -89,6 +90,10 @@ def mentor
       @mentor = (User.find(mentor_id) if mentor_id.present? && mentor_id != "later")
     end
 
+    def start_term_options
+      @start_term_options ||= %w[spring_2022 summer_2022]
```

Comment:
My view is that the applicable values here belong in the model. What was the motivation for moving them out?

Reply:
Ok, that's the interesting bit. We have two types of values here:

  1. model values - containing all the possible values that might have been assigned to models in the past (and ideally in the future), which we probably should have format validator for.
  2. form values - containing all the possible values that can be assigned at the time of using the form.

This list was previously defined as enum on the model (and also used for translations into humanized string). We could not remove values from this enum, as this would break the existing models. As a result we had a lovely cryptic `ParticipantProfile.start_terms.drop(1).each` here to skip the first option when displaying the form, which would shortly turn into `drop(2)`. This would not scale at all - in few years that enum would grow to 20 and more elements while updating the magic number in the form. In the ideal scenario those values would be generated rather than hardcoded, but I did not want to bring too much into a bug-fix PR.